python/ray/tune/config_parser.py

```diff
@@ -142,6 +143,8 @@ def to_argv(config):
             argv.append(v)
         elif isinstance(v, bool):
             pass
+        elif callable(v):
+            argv.append(json.dumps(v, cls=TuneFunctionEncoder))
```

Comment:
This seems a bit hacky but it works. Generally we might want to refactor the way from config to arvg to config